src/lib/config/defaultConfig.js

```diff
@@ -38,6 +38,28 @@ export const APP_CONFIG = {
     { name: 'contact', route: '/contact', apiURL: '2' },
   ],
   supportEmail: 'info@inveniosoftware.org',
+  environments: [
+    {
+      name: 'dev',
+      display: {
+        text: 'Dev environment',
+        color: 'blue',
+        icon: 'code',
+      },
+    },
+    {
+      name: 'qa',
```

Comment:
In CDS-ILS this will be called `sandbox`, and prod not sure yet between `prod` and `production`. If you keep `qa` here, please create a PR in CDS-ILS to override the config

Reply:
If we go with the red maybe we can take a bit less eyes hurting shade? :)

Reply:
Will update `qa` to `sandbox`. For the naming I would personally go for `production` and `development`, to match `sandbox`.